packages/cadl-ranch-specs/http/azure/client-generator-core/client-format/duration/main.tsp

```diff
@@ -0,0 +1,90 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+@doc("Test for clientFormat decorator on duration.")
+@supportedBy("dpg")
+@scenarioService("/azure/client-generator-core/client-format/duration")
+namespace _Specs_.Azure.ClientGenerator.Core.ClientFormat.Duration;
```

Comment:
Just curious why we have _Specs_ as part of the namespace. It's in some tests and not in [others](https://github.com/Azure/cadl-ranch/blob/main/packages/cadl-ranch-specs/http/authentication/api-key/main.tsp#L9).

Reply:
The namespace for cadl ranch test is always corresponding to the test folder structure. For test under `azure`, the folder structure is always the same as the core lib namespace segments, so I guess `_Specs_` is added to prevent any possible collision.

Reply:
yeah the problem is if we keep it then it conflict with the Azure.Core or other azure library namespace and then having `@scenarioService` will make it think all th eoperations inside of Azure.Core should be scenarios